docs/guides/index.rst

```diff
@@ -1,21 +1,21 @@
-How to guides
+How-To Guides
 #############
 
 .. toctree::
 
-   check-sssd-functionality
-   using-roles
-   using-ssh
+   skipping-conditional-tests
    testing-authentication
    testing-autofs
    testing-dbus
    testing-gpo
    testing-identity
+   testing-ipa-trust
    testing-ldap-krb5
+   testing-local-overrides
+   testing-local-users
    testing-netgroups
-   testing-sss_override
    testing-offline
    testing-passkey
    testing-pam
-   testing-ipa-trust
-   testing-local-users
+   using-roles
+   using-ssh
```

Comment:
Please, use four spaces to indent these lines. This will get rid of the  sphinx error.

```
<generated text>:4: ERROR: Unexpected indentation.
```

Reply:
That error message is awful, it was complaining about the lines in testing-infopipe and testing-pam

```
(.venv)  ✘ dlavu@rocket  ~/git/sssd-test-framework/docs   tox  make html 'SPHINXOPTS=-W --keep-going' 
Running Sphinx v8.0.2
loading translations [en]... done
making output directory... done
[autosummary] generating autosummary for: api.rst, api/sssd_test_framework.fixtures.rst, api/sssd_test_framework.hosts.ad.rst, api/sssd_test_framework.hosts.base.rst, api/sssd_test_framework.hosts.client.rst, api/sssd_test_framework.hosts.ipa.rst, api/sssd_test_framework.hosts.kdc.rst, api/sssd_test_framework.hosts.keycloak.rst, api/sssd_test_framework.hosts.ldap.rst, api/sssd_test_framework.hosts.nfs.rst, ..., guides/testing-netgroups.rst, guides/testing-offline.rst, guides/testing-pam.rst, guides/testing-passkey.rst, guides/using-roles.rst, guides/using-ssh.rst, index.rst, marks.rst, running-tests.rst, writing-tests.rst
loading intersphinx inventory 'pytest_mh' from https://pytest-mh.readthedocs.io/en/latest/objects.inv ...
building [mo]: targets for 0 po files that are out of date
writing output... 
building [html]: targets for 65 source files that are out of date
updating environment: [new config] 65 added, 0 changed, 0 removed
reading sources... [100%] writing-tests
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
copying assets... 
copying static files... done
copying extra files... done
copying assets: done
writing output... [100%] writing-tests
generating indices... genindex py-modindex done
writing additional pages... search done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in _build/html.
```